autowebcompat/utils.py

```diff
@@ -188,13 +188,16 @@ def make_infinite(gen_func, elems):
 
 
 def read_labels(file_name='labels.csv'):
-    try:
-        with open(file_name, 'r', newline='') as f:
-            next(f)
-            reader = csv.reader(f)
-            labels = {row[0]: row[1] for row in reader}
-    except FileNotFoundError:
+    if os.stat(file_name).st_size == 0:
         labels = {}
+    else:
+        try:
```

Comment:
Don't catch the exception here, let it fail with FileNotFoundError if the labels file doesn't exist